codespell_lib/data/dictionary.txt

```diff
@@ -1822,6 +1829,7 @@ brance->branch, branches,
 brane->brain
 braodcasted->broadcasted
 Brasillian->Brazilian
+bre->be
```

Comment:
Add brie?

Reply:
I like the idea to add typos in real case, not finding random words as scarabeo...
Also because with just one fix, it's easier to apply "-w".
Anyway, no problem, I add it.

Reply:
I'm not quite sure what you mean by "scarabeo", I've never tried the -w option, but I'd hope it still prompts you and picks the default? Certainly it strikes me there are enough potential typos in the list to make a fully automated replacement cause as many issues as it fixes.